extensions/amp-story/1.0/amp-story.js

```diff
@@ -1455,6 +1461,36 @@ export class AmpStory extends AMP.BaseElement {
     }
   }
 
+  /**
+   * @private
+   */
+  openOpacityMask_() {
+    if (!this.maskElement_) {
+      const maskEl = this.win.document.createElement('div');
+      maskEl.classList.add('i-amphtml-story-opacity-mask');
+      maskEl.addEventListener('click', () => {
+        const actions = Services.actionServiceForDoc(this.element);
+        if (this.sidebar_) {
+          actions.execute(this.sidebar_, 'close', /* args */ null,
+              /* source */ null, /* caller */ null, /* event */ null,
+              ActionTrust.HIGH);
+        }
+      });
+      this.element.appendChild(maskEl);
+      this.maskElement_ = maskEl;
+    }
+    toggle(this.maskElement_, /* display */true);
+  }
+
+  /**
+   * @private
+   */
+  closeOpacityMask_() {
+    if (this.maskElement_) {
+      toggle(this.maskElement_, /* display */false);
```

Comment:
I think the correct API for this is now `this.mutateElement` because we have deprecated `vsync` everywhere. In fact stories is one of the last extensions still on it.